### DISCUSSION TITLE
`ChangeRendererConfirmation` dialog doesn't pop up but should

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#24420] [Q&A] [answered]
Previous release:

https://github.com/ppy/osu/assets/86934170/99ade8ec-5d5d-46a9-b9b2-f6e1a9359e79

Actual release:

https://github.com/ppy/osu/assets/86934170/7ff444e2-3e1d-4c2f-b16d-f8b4fa3674d8

**Comment by bdach** [comment, marked as answer]
Please read changelog. This is intentional.

![1690719591](https://github.com/ppy/osu/assets/20418176/6ddb64e5-dae2-4627-bc1d-545d131dddd2)
